tests/libs/common/crt_error.h

```diff
@@ -0,0 +1,35 @@
+// Copyright (c) eBPF for Windows contributors
+// SPDX-License-Identifier: MIT
+#pragma once
+
+#include "catch_wrapper.hpp"
+
+#include <windows.h>
+#include <catch2/reporters/catch_reporter_event_listener.hpp>
+#include <catch2/reporters/catch_reporter_registrars.hpp>
+#include <crtdbg.h>
+
+/**
+ * @brief A catch2 listener that configures CRT error reporting.
+ */
+class _crtErrorReporter : public Catch::EventListenerBase
+{
+  public:
+    using Catch::EventListenerBase::EventListenerBase;
+
+    void
+    testRunStarting(Catch::TestRunInfo const&) override
+    {
+        // Write assertions to stderr (in addition to the defaults).
+        int mode = _CrtSetReportMode(_CRT_ASSERT, _CRTDBG_REPORT_MODE);
+        mode |= _CRTDBG_MODE_FILE;
```

Comment:
Does mode here include _CRTDBG_MODE_WNDW? If so this will cause CI/CD to hang.

Reply:
Not sure: 

> the following defaults are in effect:
> Assertion failures and errors are directed to a debug message window.
> Warnings from Windows applications are sent to the debugger's output window.
> Warnings from console applications aren't displayed.

https://learn.microsoft.com/en-us/cpp/c-runtime-library/reference/crtsetreportmode?view=msvc-170#remarks

I'm guessing the tests are a console application?

Reply:
"Assertion failures and errors are directed to a debug message window." I think this means it becomes a modal dialog box with "abort", "retry", or "ignore" buttons, which will block the CI/CD.